setup.py

```diff
@@ -32,31 +29,27 @@
 
 See http://propka.org/ for the PROPKA web server.
 """,
-      author="Jan H. Jensen",
-      author_email="jhjensen@chem.ku.dk",
-      license="LGPL v2.1",
-      url="http://propka.org",
-      keywords="science",
-      classifiers=[
-          'Development Status :: 6 - Mature',
-          'Environment :: Console',
-          'Intended Audience :: Science/Research',
-          'License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)',
-          'Operating System :: POSIX',
-          'Programming Language :: Python',
-          'Topic :: Scientific/Engineering :: Bio-Informatics',
-          'Topic :: Scientific/Engineering :: Chemistry',
-      ],
-      packages=find_packages(exclude=['scripts']),
-      package_data = {'propka': ['*.dat', '*.cfg']},
-      #scripts = ["scripts/propka31.py"],  # use entry point below
-      entry_points = {
-        'console_scripts': [
-            'propka31 = propka.run:main',
-            ],
-        },
-      zip_safe=True,
-      python_requires='>=3.5',
-      tests_require=["pandas", "numpy"],
-      test_suite="tests",
-)
+    author="Jan H. Jensen",
+    author_email="jhjensen@chem.ku.dk",
+    maintainer="Nathan Baker",
+    maintainer_email="nathanandrewbaker@gmail.com",
+    license="LGPL v2.1",
+    url="http://propka.org",
+    keywords="science",
+    classifiers=[
+        'Development Status :: 6 - Mature',
+        'Environment :: Console',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)',
+        'Operating System :: POSIX', 'Programming Language :: Python',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+        'Topic :: Scientific/Engineering :: Chemistry',
+        ],
+    packages=find_packages(exclude=['scripts']),
+    package_data={'propka': ['*.dat', '*.cfg', '*.json']},
+    entry_points={'console_scripts': ['propka3 = propka.run:main', ]},
+    zip_safe=True,
+    python_requires='>=3.5',
```

Comment:
See #63 
```suggestion
   python_requires='>=3.6',
```

Reply:
Fixed in 11bc32e9fcea71bec87681aab87ff8d52dde49b1